client/src/controllers/TooltipController.test.js

```diff
@@ -0,0 +1,137 @@
+import { Application } from '@hotwired/stimulus';
+import { TooltipController } from './TooltipController';
+
+describe('TooltipController', () => {
+  let application;
+
+  beforeEach(async () => {
+    document.body.innerHTML = `
+<section>
+  <button
+    type="button"
+    id="tooltip-default"
+    data-controller="w-tooltip"
+    data-w-tooltip-content-value="Extra content"
+  >
+    CONTENT
+  </button>
+  <button
+    id="tooltip-custom"
+    type="button"
+    data-controller="w-tooltip"
+    data-w-tooltip-content-value="Tippy top content"
+    data-w-tooltip-placement-value="top"
+    data-action="custom:show->w-tooltip#show custom:hide->w-tooltip#hide"
+  >
+    CONTENT
+  </button>
+  <button id="button" type="button">BUTTON</button>
```

Comment:
As far as I can see this isn’t used in any tests.

```suggestion
  </button>
```

Reply:
Thanks. Yeah we can remove